Dockerfile

```diff
@@ -1,27 +1,34 @@
-# Dockerfile
 # ---- Base Node ----
-FROM node:20.11.0 AS base
-ENV NEXT_TELEMETRY_DISABLED 1
-WORKDIR /app
-COPY apps/web-staking/package.json ./apps/web-staking/
-COPY apps/web-staking/package-lock.json ./apps/web-staking/
-RUN npm install --loglevel verbose -dd --prefix ./apps/web-staking
-
-# ---- Build ----
-FROM base AS build
-ENV NEXT_TELEMETRY_DISABLED 1
-WORKDIR /app
-COPY . .
-RUN npm run build --prefix ./apps/web-staking
-
-# --- Release ----
-FROM gcr.io/distroless/nodejs20-debian11 AS release
-ENV NEXT_TELEMETRY_DISABLED 1
-WORKDIR /app
-COPY --from=build /app/apps/web-staking/.next ./.next
-COPY --from=build /app/apps/web-staking/public ./public
-COPY --from=base /app/apps/web-staking/node_modules ./node_modules
-COPY apps/web-staking/package.json .
-COPY apps/web-staking/package-lock.json .
-EXPOSE 3000
-CMD ["./node_modules/next/dist/bin/next", "start"]
+    FROM node:20.11.0 AS base
+    
+    # Install PNPM
+    RUN npm install -g pnpm
+    
+    # ---- Build ----
+    FROM base AS build
+    ENV NEXT_TELEMETRY_DISABLED 1
+    WORKDIR /app
+    COPY . .
+    
+    # Install dependencies with PNPM
+    RUN pnpm install
+    
+    # Remove ESLint config file if needed
+    RUN rm .eslintrc.js
+    
+    # Build the project
+    RUN pnpm -filter @sentry/web-staking run build
+    
+    # ---- Release ----
+    FROM node:20.11.0 AS release
+    WORKDIR /app
+    
+    # Copy built project from build stage
+    COPY --from=build /app /app
+    
+    # Expose the desired port
+    EXPOSE 3000
+    
+    # Command to run the application
+    CMD ["./node_modules/next/dist/bin/next", "start"]
```

Comment:
@XDapps I don't think this can still be the right way to run the container.
Before we copied the staking app into `/app` of the container directly.
Now we have the whole monorepo built in `/app`

So this would have to be something like `.apps/web-staking/node_modules/next/dist/bin/next`.

This needs to run when running the built image with 

`docker run --env-file ./env. -p 3000:3000 [image-name]`